starter-code/index.html

```diff
@@ -1,31 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" href="css/main.css">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+<div class = background-color>
+  <button class="button"><img src="https://www.dropbox.com/s/kac3azjbqdry44g/home.png?raw=1" class = "little-house">Log in</button>
+</div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+<img src = "https://www.dropbox.com/s/brbo4hzn5gfzsb4/brand.png?raw=1" class="logo">
+<img src = "https://www.dropbox.com/s/nm2xzlssldwl13o/phones.png?raw=1" class="iphones">
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+<div class = content>
+  <h2 class = "content-title"> Capture and Share the World's Moments </h2>
+  <div class="content-text">
+  <p> Instagram is a <span>fast, beautiful</span> and <span>fun</span> way to share your life with friends and family.</p>
 
-  Oh yeah, did we mention it's free?
+  <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world. </p>
 
-  Download on the App Store
-  Get it on Google Play
+  <p>Oh yeah, did we mention it's free?</p>
+  <img src="https://www.dropbox.com/s/ntgcdfgj8r9osvt/badge-iphone.png?raw=1" class="app-store">
+  <img src="https://www.dropbox.com/s/a5flqcaqze6qij5/badge-android.png?raw=1">
 
-  About Us
-  Support
-  Blog
-  Press
+  </div>
+</div>
+<!-- <div class="button">
+    <button type="button">Log in</button>
+</div> -->
+
+</body>
+<footer>
+ 
+  <span>
```

Comment:
Since we want clickable links here we can use `<ul>` and `<li>` and `<a>` tags which we style so they're capitalized and have blue color, and on `ul` and `li` we put `display: inline; ` property.